deno_dist/jsx/intrinsic-elements.ts

```diff
@@ -0,0 +1,605 @@
+  name?: string | undefined
+  required?: boolean | undefined
+  size?: number | undefined
+  value?: string | ReadonlyArray<string> | number | undefined
+  onchange?: string | undefined
+}
+
+interface SourceHTMLAttributes extends HTMLAttributes {
+  height?: number | string | undefined
+  media?: string | undefined
+  sizes?: string | undefined
+  src?: string | undefined
+  srcset?: string | undefined
+  type?: string | undefined
+  width?: number | string | undefined
+}
+
+interface StyleHTMLAttributes extends HTMLAttributes {
+  media?: string | undefined
+  scoped?: boolean | undefined
+  type?: string | undefined
+}
+
+interface TableHTMLAttributes extends HTMLAttributes {
+  align?: 'left' | 'center' | 'right' | undefined
+  bgcolor?: string | undefined
+  border?: number | undefined
+  cellpadding?: number | string | undefined
+  cellspacing?: number | string | undefined
+  frame?: boolean | undefined
+  rules?: 'none' | 'groups' | 'rows' | 'columns' | 'all' | undefined
+  summary?: string | undefined
+  width?: number | string | undefined
+}
+
+interface TextareaHTMLAttributes extends HTMLAttributes {
+  autocomplete?: string | undefined
+  cols?: number | undefined
+  dirname?: string | undefined
+  disabled?: boolean | undefined
+  form?: string | undefined
+  maxlength?: number | undefined
+  minlength?: number | undefined
+  name?: string | undefined
+  placeholder?: string | undefined
+  readonly?: boolean | undefined
+  required?: boolean | undefined
+  rows?: number | undefined
+  value?: string | ReadonlyArray<string> | number | undefined
+  wrap?: string | undefined
+  onchange?: string | undefined
+}
+
+interface TdHTMLAttributes extends HTMLAttributes {
+  align?: 'left' | 'center' | 'right' | 'justify' | 'char' | undefined
+  colspan?: number | undefined
+  headers?: string | undefined
+  rowspan?: number | undefined
+  scope?: string | undefined
+  abbr?: string | undefined
+  height?: number | string | undefined
+  width?: number | string | undefined
+  valign?: 'top' | 'middle' | 'bottom' | 'baseline' | undefined
+}
+
+interface ThHTMLAttributes extends HTMLAttributes {
+  align?: 'left' | 'center' | 'right' | 'justify' | 'char' | undefined
+  colspan?: number | undefined
+  headers?: string | undefined
+  rowspan?: number | undefined
+  scope?: string | undefined
+  abbr?: string | undefined
+}
+
+interface TimeHTMLAttributes extends HTMLAttributes {
+  datetime?: string | undefined
+}
+
+interface TrackHTMLAttributes extends HTMLAttributes {
+  default?: boolean | undefined
+  kind?: string | undefined
+  label?: string | undefined
+  src?: string | undefined
+  srclang?: string | undefined
+}
+
+interface VideoHTMLAttributes extends MediaHTMLAttributes {
+  height?: number | string | undefined
+  playsinline?: boolean | undefined
+  poster?: string | undefined
+  width?: number | string | undefined
+  disablepictureInpicture?: boolean | undefined
+  disableremoteplayback?: boolean | undefined
```

Comment:
```suggestion
  disableRemotePlayback?: boolean | undefined
```

I believe this one is also supposed to remain camel case